setup.py

```diff
@@ -2,6 +2,8 @@
 
 from __future__ import absolute_import, division, print_function
 
+import setuptools
```

Comment:
That doesn't seem to be true, since if you install blaze from source you get the `blaze-server` script and it behaves as expected. Are you talking about tarball builds only?